scripts/github-status/github-status.rb

```diff
@@ -116,7 +116,8 @@ def get_pull_requests(state, status = [])
       pulls.partition do |p|
         next true unless @repository == "SUSE-Cloud/automation"
         pf = @client.pull_request_files(@repository, p[:number]).select do |f|
-          f[:filename] =~ %r{scripts/(mkcloud|qa_crowbarsetup\.sh|lib/.*)$}
+          f[:filename] =~ %r{scripts/(mkcloud|qa_crowbarsetup\.sh|lib/.*)$} ||
+          f[:filename] =~ %r{scripts/jenkins/log-parser/}
```

Comment:
This will still trigger an mkcloud run in the trigger job.
We need my configurable github-status (that I am building anyway for SAP) to trigger other kinds of jobs depending on the file name that matches.